build_projects/dotnet-host-build/project.json

```diff
@@ -5,13 +5,13 @@
     "emitEntryPoint": true
   },
   "dependencies": {
-    "NETStandard.Library": "1.6.1-beta-24506-02",
-    "Microsoft.NETCore.Runtime.CoreCLR": "1.1.0-beta-24507-05",
-    "Microsoft.CSharp": "4.0.1",
-    "System.Dynamic.Runtime": "4.0.11",
-    "System.Reflection.Metadata": "1.3.0",
-    "System.Runtime.Serialization.Primitives": "4.1.1",
-    "System.Xml.XmlSerializer": "4.0.11",
+    "NETStandard.Library": "1.6.2-beta-24708-03",
+    "Microsoft.NETCore.Runtime.CoreCLR": "1.2.0-beta-24708-03",
```

Comment:
@gkhanna79 I suspect this is the problem you are seeing (or related) because the CoreCLR package is not backwards compatible and this project is restoring for netcoreapp1.0 and we removed TextWriter from System.Private.CoreLib in the latest CoreCLR packages. We either need to revert back to the older version of the packages for anything that directly references CoreCLR and is netcoreapp1.0 tfm.